<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#MainWindow
{
background-image: url(:/image/res/background-new.png);
}</string>
  </property>
  <widget class="QPushButton" name="CloseBtn">
   <property name="geometry">
    <rect>
     <x>832</x>
     <y>5</y>
     <width>18</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#CloseBtn:hover
{
    border-image: url(:/image/res/close3.png);
}

QPushButton#CloseBtn:!hover
{
   border-image: url(:/image/res/close2.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="MinimizedBtn">
   <property name="geometry">
    <rect>
     <x>802</x>
     <y>5</y>
     <width>18</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#MinimizedBtn:hover
{
    border-image: url(:/image/res/mini3.png);
}

QPushButton#MinimizedBtn:!hover
{
   border-image: url(:/image/res/mini2.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>90</y>
     <width>651</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="SetupPage">
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <widget class="QWidget" name="SetUpBgWidget" native="true">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>611</width>
       <height>441</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#SetUpBgWidget
{
background-color: rgba(255, 255, 255, 245);
border-radius:20px;
}</string>
     </property>
     <widget class="QGroupBox" name="NameListGroupBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>551</width>
        <height>191</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="title">
       <string>参与名单</string>
      </property>
      <widget class="QPlainTextEdit" name="NameListPlainTextEdit">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>461</width>
         <height>101</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:white;
border:1px solid gray;
border-radius:2px;</string>
       </property>
       <property name="plainText">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="AddNameListBtn">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#AddNameListBtn
{
    
	background-color: rgb(191, 192, 202);
}</string>
       </property>
       <property name="text">
        <string> 导入... </string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>251</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>（名字请用空格分隔开）</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="SetupGroupBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>300</y>
        <width>551</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="title">
       <string>奖项设置</string>
      </property>
      <widget class="QComboBox" name="RankComboBox">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>30</y>
         <width>91</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border:1px solid gray;
border-radius:2px;
background-color:white;</string>
       </property>
       <item>
        <property name="text">
         <string>一等奖</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>二等奖</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>三等奖</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>四等奖</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>五等奖</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>六等奖</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>特等奖</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>32</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>奖次</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>人数</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="NumberSpinBox">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>74</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="maximum">
        <number>12</number>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="GoodLuckListPlainTextEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>30</y>
         <width>321</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white;
border:none;</string>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>标题设置</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="TitleLineEdit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>391</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit#TitleLineEdit
{
border-radius:2px solid gray;
border:1px solid gray;
}</string>
      </property>
      <property name="text">
       <string>标题：xx年xx联欢活动</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="RunPage">
    <widget class="QWidget" name="DisplayWidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>651</width>
       <height>491</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#DisplayWidget
{
background-image: url(:/image/DisplayAreaBg.png);
}</string>
     </property>
     <widget class="QLabel" name="RankInfoLabel">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>20</y>
        <width>151</width>
        <height>441</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel#RankInfoLabel
{
    border-image: url(:/image/res/namelist.png);
    color: rgb(255, 215, 0);
	padding-top:10px;
}</string>
      </property>
      <property name="text">
       <string>一等奖（0/2）</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QWidget" name="NameBgWidget" native="true">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>451</width>
        <height>431</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#NameBgWidget
{
background-color: rgba(255, 255, 255, 245);
border-radius:20px;
}</string>
      </property>
     </widget>
     <zorder>NameBgWidget</zorder>
     <zorder>RankInfoLabel</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="SetupBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#SetupBtn
{
color:rgb(255, 255, 255);
border-image: url(:/image/res/buttonBg.png);
border-radius:5px;
}

QPushButton#SetupBtn:pressed
{
padding-left:1px;
padding-top:1px;
}
</string>
   </property>
   <property name="text">
    <string>设 置</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="RunBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#RunBtn
{
color:rgb(255, 255, 255);
border-image: url(:/image/res/buttonBg.png);
border-radius:5px;
}

QPushButton#RunBtn:pressed
{
padding-left:1px;
padding-top:1px;
}</string>
   </property>
   <property name="text">
    <string>运 行</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>191</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/res/xy0 (3).png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>-20</y>
     <width>121</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/res/zgj.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>580</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/res/xy4.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>485</y>
     <width>65</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/image/res/xy0 (4).png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>871</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(192, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="StopTestBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 0);</string>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="StartTestBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 0);</string>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLabel" name="TitleLabel">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>53</y>
     <width>551</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
background-color:transparent;
color:rgb(192, 0, 0);
border:none;
}</string>
   </property>
   <property name="text">
    <string>标题：xx年xx联欢活动</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="ExecBtn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>540</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#ExecBtn
{
   border-image: url(:/image/res/buttonBg.png);
   border-radius:5px;
   color:white;
}

QPushButton#ExecBtn:pressed
{
   padding-left:1px;
   padding-top:1px;
}
</string>
   </property>
   <property name="text">
    <string>开 始</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>580</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/res/xy4.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>580</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/res/xy4.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="TestCount">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>label_6</zorder>
  <zorder>stackedWidget</zorder>
  <zorder>SetupBtn</zorder>
  <zorder>RunBtn</zorder>
  <zorder>label_10</zorder>
  <zorder>label_8</zorder>
  <zorder>MinimizedBtn</zorder>
  <zorder>CloseBtn</zorder>
  <zorder>StopTestBtn</zorder>
  <zorder>StartTestBtn</zorder>
  <zorder>TitleLabel</zorder>
  <zorder>label_9</zorder>
  <zorder>ExecBtn</zorder>
  <zorder>label_11</zorder>
  <zorder>label_12</zorder>
  <zorder>label_4</zorder>
  <zorder>TestCount</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
